boot/freeldr/freeldr/arch/i386/multiboot.S

```diff
@@ -122,7 +162,9 @@ mb4:
     mov cr0, eax
 
     /* Jump to real entry point */
-    ljmp16 0, FREELDR_BASE
+    ljmp 0, FREELDR_BASE
```

Comment:
Why this change?

Reply:
I wanted to test why IDA output of the compiled module looked different in GCC vs. MSVC build,
but I've just realized that I did read the output incorrectly (if I ask IDA to reanalize this portion of code as 16-bit instructions I get the correct `jmp far ptr 0:0F800h ; nop;` (the nop is padding) instead of what I read while keeping 32-bit instructions: `jmp far ptr 8D90h:0F800h` ; the output from the MSVC build already contained a separate 16-bit section and so the instruction appeared correctly).